scripts/deposit.js

```diff
@@ -154,4 +176,14 @@ async function getPastLogs(contract, event, { fromBlock, toBlock }) {
   }
 }
 
+function readJson(filepath) {
+  const s = fs.readFileSync(filepath)
+  try {
+    return JSON.parse(s)
+  } catch (e) {
+    e.message = `error parsing JSON string: ${e.message}\n${s}` 
+    throw e
+  }
```

Comment:
I find this a bit strange personally and feel that it shouldn't add much value compared to just not catching that error, but I haven't compared them locally to check.

Reply:
I dumps the JSON on the terminal, which the original error does not.

Reply:
Oh, I missed the `${s}`, my bad 😅